content/fr/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Completed
+category: technologie
+tags: ["fondamental"]
+---
+
+Un Ingress, que l'on pourrais traduire par "point d'entrée", est un ensemble de règles qui participe à gérer le trafic Internet depuis l'extérieur vers un conteneur ou un groupe de conteneurs s'exécutant dans un cluster.
+Il se compose de deux éléments : la ressource Ingress et le contrôleur Ingress.
```

Comment:
Je suggère "contrôleur d'Ingress" ici.
C'est traduit comme cela ici:
https://kubernetes.io/fr/docs/concepts/services-networking/ingress/

Reply:
fix in https://github.com/cncf/glossary/pull/2877/commits/906778cc4bacfa26d3bc213319d13920883a0a21